src/ResourceManager/Reservations/Commands.Reservations/help/Get-AzureRmReservation.md

```diff
@@ -55,7 +55,7 @@ Get specific `Reservation` details.
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
+Type: IAzureContextContainer
```

Comment:
You need to regenerate all the help files using the `UseFullTypeName` flag. Details here: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

Reply:
1. In your changelog, `src\ResourceManager\Reservations\ChangeLog.md`, you add an entry under the `Current Release` header.
2. Sounds good.

Reply:
Thanks. DONE.

Reply:
You didn't regenerate the help documentation using the `UseFullTypeName` flag.